remsfal-services/remsfal-platform/docker-compose.yml

```diff
@@ -17,3 +17,32 @@ services:
       - ZEEBE_BROKER_REPLICATION_FACTOR=1
     ports:
       - "26500:26500"
+  kafka-broker:
```

Comment:
Since the Kafka cluster should not only be used by the platform microservice, it would be better to add the configuration in the parent docker-compose.yml (remsfal-services/docker-compose.yml) before Grafana.